tests/unit/models/test_model_interfaces.py

```diff
@@ -288,15 +298,13 @@ def _vgp_matern(x: tf.Tensor, y: tf.Tensor) -> VGP:
         (VariationalGaussianProcess, _vgp),
     ],
 )
-def _gpr_interface_factory(
-    request,
-) -> Callable[[TensorType, TensorType, Optimizer | None], GaussianProcessRegression]:
+def _gpr_interface_factory(request: Any) -> _ModelFactoryType:
```

Comment:
tbh I gave up here. It looks like it's a [`FixtureRequest`](https://docs.pytest.org/en/latest/reference/reference.html#request) but that doesn't type check. This may be explained by
> has an optional param attribute